.github/workflows/release.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           path: build/*.zip
 
   build_linux:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
just making sure this was intentional 
runs-on: ubuntu-latest

Reply:
Yeah. The `ubuntu-20.04` runner has reached EOL, which causes that job to not run.